filecoin-proofs/src/bin/phase2.rs

```diff
@@ -1278,68 +1271,75 @@ fn main() {
-                }
-
-                // Default to using small before params, fallback to large before params is they exist.
-                let mut path_before = params_filename(
-                    proof,
-                    hasher,
-                    sector_size,
-                    &head,
-                    param_num - 1,
-                    ParamSize::Small,
-                    read_raw,
-                );
+                let log_filename = format!("{}_verify.log", path_after);
+                setup_logger(&log_filename);
 
-                if !Path::new(&path_before).exists() {
-                    let path_before_large = params_filename(
+                // Default to using small-raw before params, fallback to non-raw params if raw do
+                // not exist.
+                let path_before = {
+                    let small_raw = params_filename(
                         proof,
                         hasher,
                         sector_size,
                         &head,
-                        param_num - 1,
-                        ParamSize::Large,
-                        false,
-                    );
-                    info!(
-                        "small 'before' params not found `{}`, attempting to fall back to large 'before' params `{}`",
-                        path_before,
-                        path_before_large,
+                        param_num_after - 1,
+                        ParamSize::Small,
+                        true,
                     );
-                    if Path::new(&path_before_large).exists() {
-                        info!("large 'before' params found `{}`, falling back to large 'before' params", path_before_large);
-                        path_before = path_before_large;
+                    let small_nonraw = small_raw.trim_end_matches("_raw").to_string();
+                    let large = small_nonraw.replace("small", "large");
+
+                    if Path::new(&small_raw).exists() {
+                        info!("found small-raw 'before' params: {}", small_raw);
+                        small_raw
+                    } else if Path::new(&small_nonraw).exists() {
+                        info!("found small-nonraw 'before' params: {}", small_nonraw);
+                        small_nonraw
+                    } else if Path::new(&large).exists() {
+                        info!("found large 'before' params: {}", large);
+                        large
                     } else {
-                        panic!(
-                            "large 'before' params not found `{}`, fallback failed",
-                            path_before_large
+                        let err_msg = format!(
+                            "no 'before' params found, attempted: {}, {}, {}",
+                            small_raw, small_nonraw, large
                         );
+                        error!("{}", err_msg);
+                        panic!("{}", err_msg);
```

Comment:
Why both?

Reply:
Participant's may not be familiar with tmux commands, if an error occurs it's better to have the error in a sendable file rather than asking them to mess with tmux scolling.